src/Converter_TEST.cc

```diff
@@ -1885,10 +1885,16 @@ TEST(Converter, MuchNewerVersion)
   ASSERT_TRUE(sdf::Converter::Convert(&xmlDoc, "1.6"));
 }
 
-const std::string CONVERT_DOC_15_16 =
-  sdf::filesystem::append(PROJECT_SOURCE_PATH, "sdf", "1.6", "1_5.convert");
-const std::string CONVERT_DOC_16_17 =
-  sdf::filesystem::append(PROJECT_SOURCE_PATH, "sdf", "1.7", "1_6.convert");
+static std::string ConvertDoc_15_16()
```

Comment:
So did cpplint actually catch this?

Reply:
I meant to mention this in the description, but yes, cpplint caught the global string as a `runtime/string` error:
https://github.com/azeey/sdformat/blob/f5c8bb383fe13d2417f960afb261d626f1cbd40c/tools/cpplint.py#L5455. This helps some with the GSG violations we talked about, but it only works for strings.